resources/public/api/conf/1.0/endpoints/retrievePeriodSummary.raml

```diff
@@ -10,17 +10,17 @@ is:
   - errors.ruleTaxYearRangeInvalid
   - errors.invalidTaxYearParameter
   - errors.ruleTaxYearNotSupported
-displayName: Retrieve a Self-Employment Period Summary [test only]
+displayName: Retrieve a Self-Employment Period Summary
 description: "This endpoint enables you to retrieve a single self-employment period summary for a given identifier. A National Insurance number, business ID and period ID must be provided."
 (annotations.sandboxData): !include ../scenarios/retrieve_period_summary_scenarios.md
 (annotations.scope): "read:self-assessment"
 securedBy: [ sec.oauth_2_0: { scopes: [ "read:self-assessment" ] } ]
 queryParameters:
   taxYear:
     description: |
-        The tax year to which the data applies in the format YYYY-YY.
-        The start year and end year must not span two tax years. The parameter value must be at least 2023-24.
-        No gaps are allowed, for example, 2023-25 is not valid.
+      [test only] The tax year to which the data applies in the format YYYY-YY.
```

Comment:
`test only` can be removed from here?

Reply:
This is for the TYS work. Some of the endpoints will require a tax year as query parameter. However, it's not in production yet, so need to flag them as test only